code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -93,6 +94,8 @@
 
 /obj/item/weapon/card/id/examine(mob/user)
 	..()
+	if(registered_race && registered_race.id != "human")

```

Comment:
well if an id was generated without a species, so was it's owner right? which... is a much bigger issue that you'd potentially be sweeping under the rug.